src/main/java/de/retest/recheck/persistence/CloudPersistence.java

```diff
@@ -0,0 +1,87 @@
+package de.retest.recheck.persistence;
+
+import static de.retest.recheck.XmlTransformerUtil.getXmlTransformer;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.Paths;
+
+import org.keycloak.adapters.ServerRequest.HttpFailure;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.retest.recheck.Properties;
+import de.retest.recheck.auth.RetestAuthentication;
+import de.retest.recheck.persistence.bin.KryoPersistence;
+import de.retest.recheck.persistence.xml.XmlFolderPersistence;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+
+public class CloudPersistence<T extends Persistable> implements Persistence<T> {
+	private static final String SERVICE_ENDPOINT = "https://marvin.dev.cloud.retest.org/api/report";
```

Comment:
`dev` intended?

Reply:
It should be configurable and as default prod